src/clientversion.h

```diff
@@ -38,7 +38,7 @@
 #define DO_STRINGIZE(X) #X
 
 //! Copyright string used in Windows .rc files
-#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The " PACKAGE_NAME " Developers"
```

Comment:
As a separate issue from Jonas, are we sure the copyright name is something we want to change? I've had to deal with this as maintainer of Freicoin .. changing the product name would result in "Copyright 2009-2015 The Freicoin Core Developers" which would certainly not be okay (or legal?).

It would be nice to detect if PACKAGE_NAME is changed from the default, and if so add a another copyright string (while maintaining the hard-coded "Bitcoin Core" text).